src/contentFetcher.ts

```diff
@@ -114,6 +116,7 @@ export class ContentFetcher {
         this.configuration = {
             appId: configuration.appId,
             apiKey: apiKey,
+            defaultTimeout: configuration.defaultTimeout,
```

Comment:
I know this question has no relationship with the PR's purpose, but should not this constructor be private?

Reply:
I don't see any reason why it should be private.

Reply:
It throws an [error](https://github.com/croct-tech/sdk-js/pull/421/files/6d78cd1a2a01463c5572b0fec0cb20b74084c7e7#diff-ff95e402a8ee802e115b95c15250c3cf775a591aa5dc66f24fe08a5b998e6cf7L98), is it not an unexpected behavior for a constructor?

Reply:
There's nothing inherently wrong with throwing an error from a constructor. However, keep in mind that these classes aren't intended to be instantiated directly by users; they are meant to be accessed through standalone functions or SDK methods. Therefore, this is more of an internal implementation detail.